composer.json

```diff
@@ -32,8 +32,8 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": "~2.0.6",
-        "bower-asset/bootstrap": "~4.0.0"
+        "yiisoft/yii2": "~2.0.15",
```

Comment:
It should be OK to run 2.1 as well so I'd use `~2.0.15|~2.1.0`

Reply:
Of course, so the extension will be compatible with all the branches of yii2